### PR TITLE
feat(exporter/app): migrate to variable concept

### LINKED ISSUE #51: Migrate to new variable concept
As of Zeebe 0.15.0, the current payload concept is replaced by a variable-scope concept. There are new variable events to track when a variable is created or changed. We need to export the variable events and use them in the web-app to show the variables of a given scope.

### PR DESCRIPTION
**Pull request by saig0**
* create new table for variables
* remove payload column from element instances
* replace payload view with variable list
* show the history of a variable
* update a variable
* add/set a variable
* remove payload field from resolve incident dialog

closes #51

### CODE REVIEW

File: app/src/main/java/io/zeebe/monitor/rest/ViewController.java [hunk 1]
```diff
@@ -400,6 +395,61 @@ private WorkflowInstanceDto toInstanceDto(WorkflowInstanceEntity instance) {
...
+          variableDto.setUpdates(varUpdates);
+
+          dto.getVariables().add(variableDto);
+        });
+
+    final List<Long> activeScopes = new ArrayList<>();
+    if (!isEnded) {
+      activeScopes.add(instance.getKey());
+
+      // TODO add active scopes when a variable can be set locally
```
[the remark this answers is not in the document]
Comment (saig0) [reply app/src/main/java/io/zeebe/monitor/rest/ViewController.java]: With Zeebe 0.15.0?

[the remark this answers is not in the document]
Comment (Zelldon) [reply app/src/main/java/io/zeebe/monitor/rest/ViewController.java]: Ah no hopefully with 0.16.0 :sweat_smile:

**Comment by saig0** [comment]
There is no problem with setting a variable 😋 If you want to set a string then you need to put it into quotes. The values is JSON (which could be mentioned 😅). So, if you type a number then it is stored as number...

**Comment by Zelldon** [comment]
Haha :D would be interesting if someone else will have the same problem :sweat_smile:

**Pull request merged**